e2e/framework/helper.go

```diff
@@ -27,6 +31,30 @@ func EthToWei(ethValue int64) *big.Int {
 		new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil))
 }
 
+func GenerateKeyAndAddr(t *testing.T) (*ecdsa.PrivateKey, types.Address) {
+	t.Helper()
+	key, err := crypto.GenerateKey()
+	assert.NoError(t, err)
+	addr := crypto.PubKeyToAddress(&key.PublicKey)
+	return key, addr
+}
```

Comment:
If I remember correctly, this function is moved to `helper/tests` package.

Reply:
You're right 👍 
I was confused, because we standardized all of these helper methods on the IBFT branches, and here they're either nonexistent or in some other place.

Fixed 👍 
da0a77d